getting-started/con-configuring-dynamic-plugins-operator.adoc

```diff
@@ -0,0 +1,26 @@
+[id='con-configuring-dynamic-plugins-operator_{context}']
+= Configuring dynamic plugins with the Operator
```

Comment:
I noticed that the Admin guide had a section dedicated to dynamic plugins, but it is currently Helm-oriented: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-62/admin/#rhdh-installing-dynamic-plugins
Would it make sense to move this under this section (maybe a separate "Dynamic Plugin installation using the Operator" section)?